frontend/src/components/static_pages/about.jsx

```diff
@@ -1,24 +1,143 @@
+
+<p> The YMIM works to enhance participants’ global perspective of the world economy. Through access and exposure to the arts, healthcare advocacy, sciences, technology, industries, and international mission experiences we aim to enrich the cultural, personal, and professional aspects of each participant. </p>
+
+ <p>We listen to participants, address their concerns, and help connect them with resources to navigate life. We help participants develop life skills necessary to thrive as the masterbuilder of their future.</p>
+
+<p>Our approach includes models and scenarios of self-direction that cover a broad span on topics of visioning life goals, decision making, and self-sufficiency. Additionally, we help participants mature in accountability and leadership through peer-to-peer mentoring and workshops on the precepts and concepts of entrepreneurship, sustained employment, global community service activities and more.
+
+**Our Road to Impactful Programming**
+
+Alumnae input and feedback are the drivers of ongoing program enhancements. Our pre and post self assessments provide our guideposts to impactful programming. Participants of each age segment serve on ideation and planning committees. Participant-centered needs is how we determine where to prioritize cultural, education, life skills, and employment programming. 
+
+**Resources and Referrals**
+
+We link participants to resources and services beyond our capacity for those desiring or requiring additional support.  </p>
+{/* 
+**Join us!**
+
+[Enroll ](/enrollment/)today for the 2019 cohort. Applications are now being accepted. Qualifications are for young women, who are orphans, adoptees, and foster youth alumnae up to age 30\. Referrals and recommendations are welcome.
+
+**Ambassador Sponsorships Available**
+
+We welcome your thoughtful and generous contributions. Let us know which area(s) you are best able to support: operations, logistics, academic tutoring, curriculum planning, employment readiness, women’s healthcare advocacy, marketing, public relations, missions service trips, or fundraising.
+
+The **YMIM** welcomes your time, talent, and treasure. Every financial and in-kind gift is applied to our core programs – helping young women build resiliency, hope, and a bright future. 
+
+The Young Masterbuilders in Motion, Inc., is a 501(c)(3) charitable organization and your financial contributions are deductible to the full extent of the law.
+
+Consider becoming a [sponsor](/become-a-sponsor/) on behalf of foster alumnae today. Your recurring or one-time tax deductible gifts matters!
+
+[Contact us now!](/contact-us/) */}
+
+
+</div>
+<div className="lowermid group "> 
+   <div className="container clear">
+    <h1>Our People</h1>
+   <img className="floatimg" src="http://www.randolphcountync.gov/portals/0/Images/gray4.png?ver=2018-02-19-154405-587" />
+   <div className="floatp">
+  <h1>Kim Wright</h1>
+  <h4>Founder,Alumnae Executive Director</h4>
```

Comment:
lets clean up the html here with better nesting so its a bit easier to follow 

you can also install prettier (https://prettier.io/) to do this for you. 

`npm install prettier --save-dev` from frontend directory
then can run `make pretty` from root directory  and it should take care of it.